data/PiyushGoyani.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Piyush Goyani",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Software Engineer(Majorly Front), Occasionally Python Coder, ML Enthusiastic, Plant Based Programmer",
+  "avatar": "https://github.com/SaraJaoude.png",
+  "links": [
```

Comment:
you forgot to add tags.

example:
  "tags": ["Open Source", "Java", "React"],

Reply:
Added